domains/transfer-agreements/TransferAgreements.API/API/Data/TransferAgreementRepository.cs

```diff
@@ -14,4 +19,12 @@ public async Task<TransferAgreement> AddTransferAgreementToDb(TransferAgreement
         await context.SaveChangesAsync();
         return transferAgreement;
     }
-}
+
+    public async Task<List<TransferAgreement>> GetTransferAgreementsBySubjectId(Guid subjectId)
+    {
+        return await context.TransferAgreements
+            .Where(ta => ta.SenderId == subjectId)
+            .ToListAsync();
+    }
+
+    }
```

Comment:
The indentation of this line looks wrong

Reply:
Fixed